atom/parser.go

```diff
@@ -681,10 +681,8 @@ func (ap *Parser) parseAtomText(p *xpp.XMLPullParser) (string, error) {
 	lowerType := strings.ToLower(text.Type)
 	lowerMode := strings.ToLower(text.Mode)
 
-	if strings.HasPrefix(result, "<![CDATA[") &&
-		strings.HasSuffix(result, "]]>") {
-		result = strings.TrimPrefix(result, "<![CDATA[")
-		result = strings.TrimSuffix(result, "]]>")
+	if strings.HasPrefix(result, "<![CDATA[") {
```

Comment:
Should this use `strings.Contains` instead of `strings.HasPrefix`?

Reply:
yep, makes sense. I will change it